### PR TITLE
Adding new page for Security Settings to replace the Security Slider …

### PR DESCRIPTION
**Pull request by andpersand**
…page - the Security Slider directory will need to be removed. This update is to accompany the TB 8.5 release.

### CODE REVIEW

**Comment by andpersand** [comment]
Creating a Security Settings page to accompany the TB 8.5 release. Alternatively, we can replace the contents.lr file in the security-slider folder with this contents.lr file.

File: content/security-settings/contents.lr [hunk 1]
```diff
@@ -0,0 +1,56 @@
...
+---
+body:
+By default, Tor Browser protects your security by encrypting your browsing data.
+You can further increase your security by choosing to disable certain web features that can be used to attack your security and anonymity.
+You can do this by increasing Tor Browser's Security Settings in the shield menu.
+Increasing Tor Browser's security level will stop some web pages from functioning properly, so you should weigh your security needs against the degree of usability you require.
+
+##### Accessing the Security Settings
+
+<img class="col-md-6" src="../../static/images/slider.png">
```
Comment (emmapeel2) [inline content/security-settings/contents.lr:19]: This screenshot should be replaced before going live

File: content/security-settings/contents.lr [hunk 2]
```diff
@@ -0,0 +1,56 @@
...
+---
+seo_slug: security-settings
+---
+key: 8
+---
+body:
+By default, Tor Browser protects your security by encrypting your browsing data.
+You can further increase your security by choosing to disable certain web features that can be used to attack your security and anonymity.
+You can do this by increasing Tor Browser's Security Settings in the shield menu.
+Increasing Tor Browser's security level will stop some web pages from functioning properly, so you should weigh your security needs against the degree of usability you require.
```
Comment (emmapeel2) [inline content/security-settings/contents.lr:15]: Maybe we can link here to some of the documents we have to help people assess their threat model?

**Review by emmapeel2 — CHANGES_REQUESTED** [review]
I have added it to staging for now, have a look at https://lektor-staging.torproject.org/tor-browser-manual/staging/en-US/security-settings/

Please see how I added the topic to the sidebar with 7bd6d39a82d1adb738ea84bf2b4b4da1d9b4540e 

I love the perfectly trimmed lines, thanks for taking l10n into account!

**Pull request merged**